x-pack/plugins/code/public/components/main/content.tsx

```diff
@@ -101,21 +101,6 @@ const Title = styled(EuiTitle)`
 `;
 
 class CodeContent extends React.PureComponent<Props> {
-  public buttonOptions = [
-    {
-      id: ButtonOption.Code,
-      label: ButtonOption.Code,
-    },
-    {
-      id: ButtonOption.Blame,
-      label: ButtonOption.Blame,
-    },
-    {
-      id: ButtonOption.History,
-      label: ButtonOption.History,
-    },
-  ];
-
   public rawButtonOptions = [{ id: 'Raw', label: 'Raw' }];
```

Comment:
![image](https://user-images.githubusercontent.com/1646270/55924938-a3b66980-5bc0-11e9-8288-6ab097919659.png)

could you do like github, when in a binary file, use download instead of raw?

Reply:
Sure.